documentation/docs/dotenv-usage.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: dotenv-usage
-title: Enviroment Variables(.env)
-sidebar_label: Enviroment Variables
+title: Environment Variables(.env)
+sidebar_label: Environment Variables
 ---
 
 
 
 This boilerplate uses dotenv plugin to expose environment variables to the Next.js runtime configuration.
 
-Dotenv is a zero-dependency module that loads environment variables from a .env file into process.env.
+Dotenv is a zero-dependency module that loads environment variables from a `.env` file into `process.env`.
 
 
 There are two modes for use:
 
 - client: You can access both client and server.
-Example:  ```API_URL``` 
+Example:  `API_URL`
 
 - server: Only serverside can access
-Example: ```ANY_SECRET_TOKEN```
+Example: `ANY_SECRET_TOKEN`
 
-Add environment-specific variables on new lines in the form of NAME=VALUE in .env file.
+Add environment-specific variables on new lines in the form of `NAME=VALUE` in .env file.
 
 ```
 API_KEY=test1234
 ```
- İf you use unit testing, you should add same variable into .env.test file.
+ If you use unit testing, you should add same variable into `.env.test` file.
```

Comment:
If you need these variables, you will need to add them to `.env.test` file as well.